### DISCUSSION TITLE
Multi camera and mic using in android ant media sdk.

### DISCUSSION THREAD

**Discussion by yogeshkaharmis** [ant-media/Ant-Media-Server#6139] [General]
hello
I want to use Multi camera and multi mic using in android ant media sdk. but I am not getting any information about switching camera and switching mic in streaming ON. old sdk have switchCamera() method but in this we have only two mode (REAR,FRONT). 
Please give me information about work.
Thank you.

**Comment by USAMAWIZARD** [comment]
Hi Yogesh,
Please take a look at [this example](https://github.com/ant-media/WebRTC-Android-SDK/blob/master/webrtc-android-sample-app/src/main/java/io/antmedia/webrtc_android_sample_app/advanced/USBCameraActivity.java) , here we have option to switch between multiple cameras even usb cameras.

**Reply by yogeshkaharmis** [reply]
hi @USAMAWIZARD 
I will check and implement camera functionality. but one more function I want is switch microphone.
how to achieve.

**Reply by USAMAWIZARD** [reply]
let me check this and get back to you.

**Reply by USAMAWIZARD** [reply]
@yogeshkaharmis , this feature is not available on sample application , but I have created new feature request on behalf of you which is available [here](https://github.com/ant-media/Ant-Media-Server/issues/6181).
